pkgs/tools/system/netdata/default.nix

```diff
@@ -36,6 +36,8 @@ in stdenv.mkDerivation rec {
     ++ optionals withSsl [ openssl.dev ];
```

Comment:
We need better names for the options!

Reply:
Honestly can't think of a good reason why you'd ever build this without SSL, but I'm going to leave that for a future PR

Reply:
Also the .dev can be removed.